### DISCUSSION TITLE
It is possible to get the name/icon or at least the id of the wallet that the user connected

### DISCUSSION THREAD

**Discussion by victorpavlenko** [WalletConnect/walletconnect-monorepo#3804] [Developer Support] [answered]
I use **wagmi** and **web3modal** and when the user is connected, I want to show an icon for example **Uniswap**

I'll use `v3.1`

**Comment by glitch-txs** [comment, marked as answer]
yes, from active connector or useAccount

`const walletInfo = (await connector.getProvider()).session.peer`

I think it was like this 😄

**Reply by zed-wong** [reply]
Is there an example for  vue & ethers? Thanks!
